lib/src/Uncertainty/Algorithm/WeightedExperiments/LHSResult.cxx

```diff
@@ -241,6 +260,8 @@ Graph LHSResult::drawHistoryTemperature(UnsignedInteger restart, const String &
 
 Graph LHSResult::drawHistoryProbability(const String & title) const
 {
+  if (!collAlgoHistory_.getSize())
+    throw InvalidArgumentException(HERE) << "No design stored!";
```

Comment:
and here

Reply:
The argument of the method is the graph title. We are using `CollAlgoHistory_[optimalIndex_]`  via `drawHistoryProbability(UnsignedInteger restart, const String & title)`.